nautobot/extras/management/__init__.py

```diff
@@ -184,98 +269,113 @@ def create_custom_statuses(
             try:
                 # May fail with an IntegrityError if a status with a different slug has a name matching this one
                 # May fail with a FieldError if the Status model no longer has a slug field
-                obj, created = Status.objects.get_or_create(slug=slug, defaults=defaults)
+                obj, created = Metadata_Model.objects.get_or_create(slug=slug, defaults=defaults)
             except (IntegrityError, FieldError) as error:
                 # OK, what if we look up by name instead?
                 defaults = choice_kwargs.copy()
                 name = defaults.pop("name")
                 # FieldError would occur when calling create_custom_statuses after status slug removal
                 # migration has been migrated
-                # e.g nautobot.extras.tests.test_management.StatusManagementTestCase.test_populate_status_choices_idempotent
+                # e.g nautobot.extras.tests.test_management.MetadataManagementTestCase.test_populate_status_choices_idempotent
```

Comment:
Revert perhaps?

Reply:
I did change the name of the test case to `MetadataManagementTestCase`. So this is accurate right now.